optimize/self-managed/optimize-deployment/configuration/history-cleanup.md

```diff
@@ -6,10 +6,10 @@ description: "Make sure that old data is automatically removed from Optimize."
 
 To satisfy data protection laws or just for general storage management purposes, Optimize provides an automated cleanup functionality.
 
-:::note
-By default, the history cleanup is disabled in Optimize. Before enabling it, you should consider the type of cleanup and time to live period that fits to your needs. Otherwise, historic data intended for analysis might get lost irreversibly.
+:::note Note for Camunda Platform 7 users
```

Comment:
Please see below comment, as the changes in this file also might need changing accordingly

Reply:
I will add a small change about the option to enable them individually, but I think it still makes sense to keep this note specific to C7 users.